### DISCUSSION TITLE
Cylindrical particle orientation

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#12867] [Q&A] [answered]
Hi!

Does the orientation of a cylindrical Lagrangian particle affect the heat transfer coefficient via free convection as in Sec. 8.2.2 from the User Guide? 

If this is not the case, is the particle then always assumed to be horizontal or vertical? Can this be controlled?

**Comment by ericvmueller** [comment, marked as answer]
The choice of natural convection model is connected to whether `HORIZONTAL=T` on the `SURF` line associated with the class of particles. So it will be the same for all particles in a class, either vertical by default or horizontal if you set that flag, and it does not depend on any defined `ORIENTATION` of the particles.

**Reply by r-broek** [reply]
Thank you!
